src/vcBPA.cpp

```diff
@@ -1120,6 +1167,24 @@ void vcBPA_CompareExport(vcState *pProgramState, const char *pOldModelPath, cons
     ++item.attributes.count;
   }
 
+  if (addDisplacementDirection)
+  {
+    item.attributes.pDescriptors[item.attributes.count].blendMode = vdkABM_SingleValue;
+    item.attributes.pDescriptors[item.attributes.count].typeInfo = vdkAttributeTypeInfo_float32;
```

Comment:
Why not use `vdkAttributeTypeInfo_vec3f32`?

Reply:
Can't- I forgot to take it out of VDK actually.

Reply:
Ah ok - @int-Frank just ignore this comment! 😄